prow/flagutil/git.go

```diff
@@ -28,8 +29,8 @@ import (
 // GitOptions holds options for interacting with git.
 type GitOptions struct {
 	host          string
-	user          string
-	email         string
+	User          string
```

Comment:
So the use from `bumper` is not the `git` library. Although the options _seem_ the same, they are different. We should have a separate struct for them that's not tied to this, unless we expect a single binary to use both?

Reply:
I dont understand what you mean by "So the use from bumper is not the git library".
I am using both git and bumper here(https://github.com/openshift/ci-tools/pull/1211/files#diff-4440cf6f38e6026b7a8028aa83c9f664R33) 
Am I doing that wrong?

Reply:
moved this from git options into new class GitAuthorOptions in bumper.go